README.md

```diff
@@ -1,2 +1,104 @@
-# apiv3-sdk
-A software development kit for the WattTime API, showing basic examples of usage. 
+# About
+This SDK is meant to help users with basic queries to WattTime’s API (version 3), and to get data returned in specific formats (e.g., JSON, pandas, csv).
+
+Users must first [register for access to the WattTime API here](https://watttime.org/docs-dev/data-plans/).
```

Comment:
Actually, to register an account, they need to post to /register, right? Should we add that to the API?